src/Video.tsx

```diff
@@ -229,17 +229,25 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
 
       Platform.select({
         ios: () => {
-          nativeRef.current?.setNativeProps({
-            seek: {
+          VideoManager.seek(
```

Comment:
I am afraid of impact on windows regarding to typings and ...
BTW, won't it be cleaner to extract 
const callSeekFunction = () => {
          VideoManager.seek(
            {
              time,
              tolerance: tolerance || 0,
            },
            getReactTag(nativeRef),
          );

and then
Platform.select({
        ios: callSeekFunction,
        android: callSeekFunction,
....